src/audience.ts

```diff
@@ -1,9 +1,12 @@
 export default class Audience {
-    public id: number;
-    public name: string;
+    public audience_id: number;
+    public expiration_timestamp_ms?: number;
```

Comment:
Should we just remove the `expiration_timestamp_ms` field entirely since we aren't using it? Customers may think null means never expires or something when really we're just never returning anything